library/ssl_tls13_generic.c

```diff
@@ -1328,6 +1328,39 @@ int mbedtls_ssl_tls13_write_change_cipher_spec( mbedtls_ssl_context *ssl )
 
 #endif /* MBEDTLS_SSL_TLS1_3_COMPATIBILITY_MODE */
 
+/* Early Data Extension
```

Comment:
Align the name in RFC 8446
```suggestion
/* Early Data Indication Extension
```